.github/workflows/actions/prepare-distribution/action.yml

```
@@ -85,6 +85,17 @@ runs:
         Compatible systems include, but are not limited to, \`Ubuntu 20.04+\` or \`Debian 11+\` (Bullseye)).
         EOF
 
+    - name: Fetch the latest version of the unstable tag
+      shell: bash
+      run: |
+        cat >> ./release-notes-addon.txt << EOF
+        
+        ## Fetch the latest version of the \`unstable\` tag
+        The \`unstable\` tag is updated with a new commit id when a new \`unstable\` release is published.
+        To fetch the latest version of the unstable tag, execute the git commands:
+        \`git tag -d unstable && git fetch --tags\`
```

This command would fetch only the `unstable` tag:
```suggestion
        \`git tag -d unstable && git fetch origin unstable\`
```

---

I tried your suggestion but it doesn't seem to pull the tag.
This command below works, is it ok for you?
```suggestion
        \`git tag -d unstable && git fetch --tags origin unstable \`
```